glue/sales_val_flagging.py

```diff
@@ -102,12 +102,35 @@ def ptax_adjustment(df, groups, ptax_sd):
 
     group_string = "_".join(groups)
 
-    df["ptax_flag_w_deviation"] = df["ptax_flag_original"] & (
-        (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1])
-        | (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0])
-        | (df[f"sv_price_per_sqft_deviation_{group_string}"] >= ptax_sd[1])
-        | (df[f"sv_price_per_sqft_deviation_{group_string}"] <= -ptax_sd[0])
-    )
+    if condos == False:
```

Comment:
The reason the ptax flags weren't being assigned in condos is that there wasn't a conditional block that handles the different groups. This means that condo data used the res groups, resulting in na values for the conditional statements that the ptax outliers are built upon.

Reply:
**[Nitpick, non-blocking]** Makes sense! A more Pythonic check here would be to use `condos is False`, or better yet, check the truthiness of the value:

```suggestion
    if not condos:
```